modules/nw-ingress-setting-internal-lb.adoc

```diff
@@ -0,0 +1,57 @@
+// Module included in the following assemblies:
+//
+// * networking/ingress-operator.adoc
+
+[id="nw-ingress-setting-internal-lb_{context}"]
+= Configuring an Ingress Controller to use an internal load balancer
+
+When creating an Ingress Controller on cloud platforms, the Ingress Controller is published by a public cloud load balancer by default.
+As an administrator, you can create an Ingress Controller that uses an internal cloud load balancer.
+
+You can configure the `default` Ingress Controller for your cluster to be internal by deleting and recreating it.
+
+[IMPORTANT]
+====
+If you want to change the `scope` for an `IngressController` object, you must delete and then recreate that `IngressController` object. You cannot change the `.spec.endpointPublishingStrategy.loadBalancer.scope` parameter after the Custom Resource (CR) is created.
+====
+
+See the link:https://kubernetes.io/docs/concepts/services-networking/service/#internal-load-balancer[Kubernetes Services documentation]
+for implementation details.
+
+.Prerequisites
+
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create an `IngressController` Custom Resource (CR) in a file named `<name>-ingress-controller.yaml`, such as in the following example:
++
+[source,yaml]
+----
+apiVersion: operator.openshift.io/v1
+kind: IngressController
+metadata:
+  namespace: openshift-ingress-operator
+  name: <name> <1>
+spec:
```

Comment:
`.spec.domain` is required unless we're talking about the `default` resource (which can omit `.spec.domain` and claim the default domain).

Reply:
@ironcladlou I'm helping out to get this PR merged and just want to verify - this is the only update this PR needs? Thanks!

(I'll also likely have follow-up questions related to "User defined IngressControllers at installation".)